src/doc/CreateHighlightDialog.js

```diff
@@ -83,7 +83,20 @@ class CreateHighlightDialog extends EventEmitter {
             return;
         }
 
-        if (!this.isMobile) {
+        // Select page of first node selected
+        const pageInfo = getPageInfo(selection.anchorNode);
+        if (!pageInfo.pageEl) {
+            return;
+        }
+
+        if (this.isMobile) {
+            this.pageInfo = getPageInfo(selection.anchorNode);
```

Comment:
could this mobile specific logic somehow be integrated into the `setPosition` call?